test/TestProjects/ResourceClients-LowLevel/Generated/ResourceServiceClient.cs

```diff
@@ -0,0 +1,242 @@
+#pragma warning disable AZC0002
+        public virtual Response GetParameters(RequestContext context = null)
+#pragma warning restore AZC0002
+        {
+            using var scope = _clientDiagnostics.CreateScope("ResourceServiceClient.GetParameters");
+            scope.Start();
+            try
+            {
+                using HttpMessage message = CreateGetParametersRequest(context);
+                return _pipeline.ProcessMessage(message, _clientDiagnostics, context);
+            }
+            catch (Exception e)
+            {
+                scope.Failed(e);
+                throw;
+            }
+        }
+
+        /// <summary> Get all groups. It is defined in `Group` subclient, but must be promoted to the `Service` client. </summary>
+        /// <param name="context"> The request context, which can override default behaviors on the request on a per-call basis. </param>
+#pragma warning disable AZC0002
+        public virtual async Task<Response> GetGroupsAsync(RequestContext context = null)
+#pragma warning restore AZC0002
+        {
+            using var scope = _clientDiagnostics.CreateScope("ResourceServiceClient.GetGroups");
+            scope.Start();
+            try
+            {
+                using HttpMessage message = CreateGetGroupsRequest(context);
+                return await _pipeline.ProcessMessageAsync(message, _clientDiagnostics, context).ConfigureAwait(false);
+            }
+            catch (Exception e)
+            {
+                scope.Failed(e);
+                throw;
+            }
+        }
+
+        /// <summary> Get all groups. It is defined in `Group` subclient, but must be promoted to the `Service` client. </summary>
+        /// <param name="context"> The request context, which can override default behaviors on the request on a per-call basis. </param>
+#pragma warning disable AZC0002
+        public virtual Response GetGroups(RequestContext context = null)
+#pragma warning restore AZC0002
+        {
+            using var scope = _clientDiagnostics.CreateScope("ResourceServiceClient.GetGroups");
+            scope.Start();
+            try
+            {
+                using HttpMessage message = CreateGetGroupsRequest(context);
+                return _pipeline.ProcessMessage(message, _clientDiagnostics, context);
+            }
+            catch (Exception e)
+            {
+                scope.Failed(e);
+                throw;
+            }
+        }
+
+        /// <summary> Get all groups. It is defined in `Item` subclient, but must be promoted to the `Service` client, because it has neither `groupId` nor `itemId` parameters. </summary>
+        /// <param name="context"> The request context, which can override default behaviors on the request on a per-call basis. </param>
+#pragma warning disable AZC0002
+        public virtual async Task<Response> GetAllItemsAsync(RequestContext context = null)
```

Comment:
I had thought we removed `All` from these method names in the generator?

Reply:
No, we don't. Here is the code that does replacement: https://github.com/Azure/autorest.csharp/blob/feature/v3/src/AutoRest.CSharp/Common/Utilities/StringExtensions.cs#L371